.github/workflows/deploy.yml

```diff
@@ -0,0 +1,45 @@
+name: Deploy assets to AWS S3
+
+on:
+  workflow_call:
+    inputs:
+      bucket:
+        required: false
+        type: string
+        default: answers
+      directory:
+        required: true
+        type: string
+      cache-control:
+        required: false
+        type: string
+        default: no-cache
+    secrets:
+      AWS_ACCESS_KEY_ID:
+        required: true
+      AWS_SECRET_ACCESS_KEY:
+        required: true
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download build-output artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: build-output
+          path: dist/
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
```

Comment:
did we always specify the aws region?

Reply:
Yes, the CircleCI environment variables include AWS_REGION: xxxxst-1 which I assume is us-east-1. I checked AWS and assets.sitescdn.net is in us-east-1

Reply:
I'm 99% certain that it is `us-east-1`, but I'm trying to verify the contents of that env var. I'm a bit paranoid when it comes to region stuff. In Okta, when I log into S3, it automatically puts me in the East region. So anything I look at in S3 will already be in that region. That doesn't mean those buckets don't exist in other regions though. 

I don't think AWS_REGION could be some sort of wild card that allows for East and West, but I want to make sure.